proposals/description/tip_111.ts

```diff
@@ -0,0 +1,83 @@
+import { TemplatedProposalDescription } from '@custom-types/types';
+
+const eth_lbp: TemplatedProposalDescription = {
+  title: 'TIP-111: Increase FEI Stable backing to 90-100%',
+  commands: [
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.ethLidoPCVDepositOld, // pcvDeposit
+        addresses.steth, // token
+        addresses.ethLidoPCVDeposit, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Migrate all stETH to new PCVDeposit'
+    },
+    {
+      target: 'balancerGaugeStaker',
+      values: '0',
+      method: 'unstakeFromGauge(address,uint256)',
+      arguments: (addresses) => [
+        addresses.bpt30Fei70Weth, // token
+        '252865858892972812879565' // amount
+      ],
+      description: 'Unstake all B-70WETH-30FEI from Balancer gauge'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.balancerGaugeStaker, // pcvDeposit
+        addresses.bpt30Fei70Weth, // token
+        addresses.balancerDepositFeiWeth, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Move all B-70WETH-30FEI to Balancer deposit'
+    },
+    {
+      target: 'balancerDepositFeiWeth',
+      values: '0',
+      method: 'exitPool(address)',
+      arguments: (addresses) => [addresses.balancerDepositFeiWeth],
+      description: 'Exit LP position and send WETH to self, while burning the FEI'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.balancerDepositFeiWeth, // pcvDeposit
+        addresses.weth, // token
+        addresses.aaveEthPCVDeposit, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Move all WETH to Aave PCVDeposit'
+    },
+    {
+      target: 'aaveEthPCVDeposit',
+      values: '0',
+      method: 'deposit()',
+      arguments: () => [],
+      description: 'Deposit all WETH in Aave'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'swapDeposit(address,address)',
+      arguments: (addresses) => [addresses.ethLidoPCVDepositOldWrapper, addresses.ethLidoPCVDeposit],
+      description: 'Swap old & new Lido ETH PCVDeposits in CR Oracle'
+    }
+  ],
+  description: `
+This on-chain perform will perform the following actions :
```

Comment:
Would put the FIP title in this description. 

Then small typo, "This on-chain proposal will perform..."